prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-review-status/components/review-status-content/review-status-content.component.html

```diff
@@ -12,4 +12,30 @@ <h3 class="mb-4">This Enrollee was manually adjudicated for the following reason
       </div>
     </div>
   </div>
+  <div class="col-6 pl-0">
```

Comment:
I think this ticket was looking for a way to combine previous statuses with adjudicator notes for displaying the history of statuses with descriptions like in the mockups.